css-three-column-layout/styles.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #1996da;
+  width: 80%;
+  margin: auto;
+  height: 100vh;
```

Comment:
You shouldn't set size or margin on the body like this. Instead, you should make a `div` with a `container` class to hold all your content, then apply this styling to the `container` class.